it/mail/chrome/messenger/preferences/downloads.dtd

```diff
@@ -1,24 +1,24 @@
-<!ENTITY askOnSave.label              "Attachments Folder">
-<!ENTITY alwaysAsk.label              "Ask me where to save every file">
+<!ENTITY askOnSave.label              "Cartella allegati">
+<!ENTITY alwaysAsk.label              "Chiedimi dove salvare ogni file">
 <!ENTITY alwaysAsk.accesskey          "A">
-<!ENTITY neverAsk.label               "Save all attachments to this folder:">
+<!ENTITY neverAsk.label               "Salva tutti gli allegati in questa cartella:">
 <!ENTITY neverAsk.accesskey           "S">
-<!ENTITY desktop.label                "Desktop">
+<!ENTITY desktop.label                "Scrivania">
```

Comment:
CORRECTIONS NEEDED. "Desktop" remains unchanged in Italian too. The correct entry should be: 

+<!ENTITY desktop.label                "Desktop">